sr_ur_launch/README.md

```diff
@@ -0,0 +1,138 @@
+# Starting the robots (simulated / real)
```

Comment:
This file is basically describing how to use hand E and hand E + arm.
We should modify it to also contain instructions to launch UR arm alone.